test/metabase/driver/mysql_test.clj

```diff
@@ -710,9 +710,7 @@
                                    (sql-jdbc.conn/with-connection-spec-for-testing-connection
                                      [spec [:mysql new-connection-details]]
                                      (with-redefs [sql-jdbc.conn/db->pooled-connection-spec (fn [_] spec)]
-                                       (driver/current-user-table-privileges driver/*driver*
-                                                                             (assoc (mt/db) :name "test table privileges db"
-                                                                                    :details new-connection-details))))))]
+                                       (sql-jdbc.sync/current-user-table-privileges driver/*driver* spec)))))]
```

Comment:
wdyt about a test that checks that we can call implementations with kwargs correctly. This might suffice
```suggestion
                                       (sql-jdbc.sync/current-user-table-privileges driver/*driver* spec {})))))]
```

Reply:
Good idea